service/web/tabs/app/src/components/StopAllBackfillsButton.tsx

```diff
@@ -67,8 +99,8 @@ class StopAllButton extends React.Component {
           />
         }
       />
-    );
+    )
   }
 }
 
-export default StopAllButton
\ No newline at end of file
+export default StopAllButton
```

Comment:
@adrw where does this fit in our V1 vs V2 UI? Where are we on that? Maybe we should push on finishing that?